modules/op-query-results-name.adoc

```diff
@@ -0,0 +1,81 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-results-for-openshift-pipelines-observability.adoc
+
+:_content-type: PROCEDURE
+[id="query-results-uuid_{context}"]
+= Querying for results and records by name
+
+[role="_abstract"]
+You can list and query results and records using their names.
+
+.Prerequisites
+
+* You installed {tekton-results}.
+* You installed the `opc` utility and prepared its environment to quety {tekton-results}.
+* You installed the `jq` package.
+
+.Procedure
+
+. To list the names of all results that correspond to pipeline runs and task runs created in a namespace, run the following command:
```

Comment:
Unfortunately this does not work, because it can very well be misread as "....runs created in a namespace by running the command". I just used a full stop for now but a better suggestion would be appreciated.